src/test/duke/customClass/CommandRouterTest.java

```diff
@@ -0,0 +1,16 @@
+package duke.customClass;
+
+import org.junit.jupiter.api.Test;
+import duke.customClass.Command;
+
+import static org.junit.jupiter.api.Assertions.*;
```

Comment:
Should this import statement be listed explicitly instead?
I noticed the same issue in several other places as well.